### PR TITLE
Update MacPorts install instructions

### PR DESCRIPTION
**Pull request by cary-ilm**
MacPorts is no longer out of date.

### CODE REVIEW

File: website/install.rst [hunk 1]
```diff
@@ -39,15 +39,12 @@ On macOS, install via `Homebrew <https://formulae.brew.sh/formula/openexr>`_:
...
-python bindings. ``pip install openexr`` installs the `openexrpython
-<https://github.com/jamesbowman/openexrpython>`_ module, which is not
-affiliated with the OpenEXR project or the ASWF. Please direct
-questions there.
+Alternatively, you can install on macOS via `MacPorts
+<https://ports.macports.org/port/openexr>`_:
+
+.. code-block::
+
+   % sudo port install openexr
```
Comment (meshula) [inline website/install.rst]: is sudo needed here?

Comment (meshula) [reply website/install.rst]: I don't think we install anything into a protected area.

Comment (cary-ilm) [reply website/install.rst]: I lifted that from https://ports.macports.org/port/openexr, but yes, probably better to omit it in our instructions. At the very least it should be consistent with the brew instructions. 

I also added a reference to vcpkg for Windows.

**Review by meshula — APPROVED** [review]
lgtm!

**Pull request merged**